### PR TITLE
fix: keep placeholder in multivalue select when a value exists

### PR DESCRIPTION
**Pull request by eschutho**
### SUMMARY
Some users were not noticing that they could enter multiple values in some fields, and it was also difficult to know which were single vs multiple, just by looking at the select input. With some input from others, I have updated the placeholder to indicate that multiple selections can be made, and also kept the placeholder in the input field after a selection has been made. We also talked about keeping the focus on the new input after a selection was made, but decided to put that in a separate PR.  

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Growth_Rate__vBFmtRXfi_

### TEST PLAN
unit tests added 

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [x] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/components/Select/styles.tsx [hunk 1]
```diff
@@ -272,6 +294,24 @@ export const DEFAULT_COMPONENTS: SelectComponentsConfig<any> = {
...
+    <div style={{ position: 'relative' }}>
+      <Input {...props} />
+      {!!(props.selectProps.isMulti && props.selectProps.value.length) && (
+        <span
+          style={{
+            color: '#879399',
+            position: 'absolute',
+            top: '2px',
+            left: '4px',
+            width: '100vw',
```
Comment (eschutho) [inline superset-frontend/src/components/Select/styles.tsx:307]: Looking for suggestions as to the best place to put these styles.

File: superset-frontend/src/explore/components/controls/AdhocFilterControl.jsx [hunk 2]
```diff
@@ -267,7 +267,7 @@ export default class AdhocFilterControl extends React.Component {
         <OnPasteSelect
           isMulti
           name={`select-${this.props.name}`}
-          placeholder={t('choose a column or metric')}
+          placeholder={t('choose one or more column or metric')}
```
Comment (eschutho) [inline superset-frontend/src/explore/components/controls/AdhocFilterControl.jsx:270]: I don't have strong opinions on whether this should be singular or plural "column/columns" and "metric/metrics", and it seems the grammar guidelines are loose as well.

File: superset-frontend/src/components/Select/styles.tsx (hunk 1 above)
Comment (nytai) [reply superset-frontend/src/components/Select/styles.tsx:307]: in an emotion `styled.span`?

Comment (eschutho) [reply superset-frontend/src/components/Select/styles.tsx]: ok, I put the styles in a const.. seems to be a common pattern? 
I didn't see a way to pull a color in from a pallet, though, so it's hard-coded unless there's any advice there.

File: superset-frontend/src/components/Select/styles.tsx [hunk 3]
```diff
@@ -241,9 +241,39 @@ export const DEFAULT_STYLES: PartialStylesConfig = {
   }),
 };
 
-const { ClearIndicator, DropdownIndicator, Option } = defaultComponents;
+const multiInputSpanStyle = css`
+  color: '#879399',
```
Comment (nytai) [inline superset-frontend/src/components/Select/styles.tsx:245]: if this color is part of the theme, https://github.com/apache-superset/superset-ui/blob/master/packages/superset-ui-core/src/style/index.ts, you should be able to reference it with `${({ theme }) => theme.colors...}`

Comment (eschutho) [reply superset-frontend/src/components/Select/styles.tsx:245]: good to know. Unfortunately it doesn't appear to be in the theme.

**Review by nytai — APPROVED** [review]
lgtm

File: superset-frontend/src/components/Select/styles.tsx (hunk 3 above)
Comment (nytai) [reply superset-frontend/src/components/Select/styles.tsx:245]: we could also add it to the theme if we expect reuse or want it to be themeable

**Comment by ktmud** [comment]
@eschutho @nytai The placeholder seems a little too dark for me. And it's also not a placeholder (focusing the input it does not disappear). Do you mind taking a look?

**Comment by ktmud** [comment]
I would also recommend changing the copy to "add additional metrics/filters/columns" when there are values in the input.

**Comment by ktmud** [comment]
image

Looks like the CSS broke after migrating to styled components.

**Comment by junlincc** [comment]
Thanks for pointing that out. @eschutho <https://github.com/eschutho> would
you mind fixing it?🙏 in terms of the wording, we should probably put more
thinking into it. @Srinivasa Kadamati <srini@preset.io> @Stephen Jordan
<stephan@preset.io>  any thoughts?

**Comment by eschutho** [comment]
Thanks for catching/handling @etr2460. I'm looking into this now, but feel free to revert anyway since I missed the message yesterday. I'll sync with you @junlincc on wording today.

**Comment by etr2460** [comment]
Thanks Elizabeth!

**Pull request merged**